app.py

```diff
@@ -1,11 +1,30 @@
-from flask import Flask
+from flask import Flask, render_template, request, url_for, flash, redirect
 
 app = Flask(__name__)
+app.config['SECRET_KEY'] = 'ccb0ce4501150d103e1dae7b8431d6ab70c026903df97cbd'
```

Comment:
As you mentioned this needs to be hid, we'll take a look if there is some way to do this.